packages/smart-contracts/.mocharc.json

```diff
@@ -2,5 +2,5 @@
     "extension": ["ts"],
     "spec": "test/**/*.ts",
     "require": "ts-node/register",
-    "timeout": 120000
+    "timeout": 1200000
```

Comment:
btw, why Mocha and not Jest?

Reply:
I was using `ethereum-waffle` for testing and that was using mocha so i continued using it. If you wanma change it to jest feel free to do so.

Reply:
yea maybe later